wagon-providers/pom.xml

```diff
@@ -52,24 +51,11 @@ under the License.
         <artifactId>httpclient</artifactId>
         <version>4.5.14</version>
       </dependency>
-      <!-- Overwrite transitive dependency version of HttpClient -->
-      <dependency>
-        <groupId>commons-codec</groupId>
-        <artifactId>commons-codec</artifactId>
-        <version>1.16.1</version>
-        <scope>runtime</scope>
-      </dependency>
```

Comment:
This does not apply anymore?

Reply:
i see no longer reports of active issues with latest version of httpclient